README.md

```diff
@@ -19,14 +19,17 @@ Neural network-based singing voice synthesis library for research.
 ## Installation
 
 - Python 3.6 or newer
-- [nnmnkwii](https://github.com/r9y9/nnmnkwii): **development** version (master branch) is required
-- [pysinsy](https://github.com/r9y9/pysinsy) **development** version is required. Please have a look at the repostiory for installation.
 - Pytorch >= 1.x
```

Comment:
Could you revert this change? I would like to keep the current manual installation policy for maximum flexibility.